docs/prod-auth-first-login.md

```diff
@@ -1,5 +1,4 @@
-Authentication and logging into Zulip the first time
```

Comment:
This applies to the git commit message: use imperative voice, so say "Improve", not "Improves.  I'd also quote "Zulip in Production."